components/database/mariadb-101/mysql-common.p5m

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
We need only one mysql-common package, and it's currently is provided by mariadb 5.5 component